swagger/src/main/scala/org/http4s/rho/swagger/SwaggerSupport.scala

```diff
@@ -48,5 +43,22 @@ object SwaggerSupport {
     val sb = new SwaggerModelsBuilder(swaggerFormats)
     routes.foldLeft(Swagger())((s, r) => sb.mkSwagger(apiInfo, r)(s))
   }
+
+  /**
+   * Create a RhoService with the route to the Swagger json
+   * for the given Swagger Specification
+   */
+  def createSwaggerRoutes(
```

Comment:
I think returning `RhoService` is the best option to maniupulate them as the same just as I did in the test case.